README.md

```diff
@@ -260,6 +260,7 @@ Neovim supports a wide variety of UI's.
 
 Tree-sitter is a new system coming in Neovim 0.5 that incrementally parses your code into a tree that works, even with errors in your syntax. These colorschemes have specifically set colors for Tree-sitter highlight groups. Vim colorschemes will work with the new groups out of the box.
 
+- [shaeinst/roshnivim-cs](https://github.com/shaeinst/roshnivim-cs) - Colorscheme for (neo)vim written in lua, specially made for roshnivim with Tree-sitter support.
```

Comment:
Hi, thanks for the PR! `neovim` should be written `Neovim` and `lua` as `Lua`. If you apply those changes I can merge the PR!